### PR TITLE
keys import

### PR DESCRIPTION
**Pull request by NikVolf**
by filename `parity account import ~/my_keys/mykeyfile1.json`
by directory `parity account import ~/my_testnet_keys_dir`
allows multiple entries

### CODE REVIEW

File: util/src/keys/geth_import.rs [hunk 1]
```diff
@@ -80,28 +89,62 @@ pub fn import_geth_key(secret_store: &mut SecretStore, geth_keyfile_path: &Path)
...
+	// check if it is just one file or directory
+	if let Ok(meta) = fs::metadata(path) {
+		if meta.is_file() {
+			try!(import_geth_key(secret_store, Path::new(path)));
+			return Ok(1);
+		}
+		else if meta.is_dir() {
+			let mut path_buf = PathBuf::new();
+			path_buf.push(path);
+			path_buf.push("*.*");
```
Comment (gavofyork) [inline util/src/keys/geth_import.rs]: this is a weird way of doing things isn't it?

why not just accept a list of files and let the shell do the globbing?

Comment (NikVolf) [inline util/src/keys/geth_import.rs]: @gavofyork 
windows?

Comment (gavofyork) [inline util/src/keys/geth_import.rs]: supporting paths is fine. windows has powershell if users are desperate to use wildcards.

**Comment by gavofyork** [comment]
so i'd really prefer to keep MS-DOS-era globbing out of the codebase.

it should really be:

- any mix of filenames and directories, e.g.:
- `parity account import ~/my_keys/mykeyfile1.json ~/my_keys/mykeyfile2.json`
- `parity account import ~/my_testnet_keys_dir`
- `parity account import ~/my_testnet_keys_dir ~/my_keys/mykeyfile.json`

**Comment by NikVolf** [comment]
@gavofyork done

**Pull request merged**